lifterlms/courses-lessons/exclude-posts-from-user-roles.php

```diff
@@ -0,0 +1,52 @@
+/**
+ * Exclude posts (given their IDs) from being seen by users given a list of roles.
+ * 
+ * Learn more at: https://lifterlms.com/link-to-content-if-available-or-remove-this-line/
+ * 
+ * You can add this recipe to your site by creating a custom plugin
+ * or using the Code Snippets plugin available for free in the WordPress repository.
+ * Read this companion documentation for step-by-step directions on either method.
+ * https://lifterlms.com/docs/adding-custom-code/
+ */
+
+function exclude_posts_for_roles($query) {
+    // Do not exclude when the user is the WordPress administrator
+    if (is_admin()) {
+        return;
+    }
+
+    // Replace the array with your actual roles you want
+    $roles_to_exclude = array(
+		'student',
+		'editor'
```

Comment:
Can we add a comma at the end of this line (and the last line of the array below)? This is part of the standards, but also lessens the chance of errors if they're adding more elements after the last one and not realizing they need to add a comma here.